.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,29 @@
+name: Emscripten build
+on: [push, pull_request]
+concurrency:
+  group: ${{ github.workflow }}-${{ github.job }}-${{ github.ref }}
+  cancel-in-progress: true
+jobs:
+  test:
+    runs-on: ubuntu-latest
+ 
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - uses: mamba-org/setup-micromamba@v1
+        with:
+            environment-name: xsimd
+            create-args: >-
+                microsoft::playwright
+                python
```

Comment:
out of curiosity, why `python` there?

Reply:
python is need for playright which is a python package allowing you to start a (headless) browser and interact with it.